themes/mongodb/src/css/feedback.css

```diff
@@ -43,6 +43,10 @@
     width: 100%; }
   .deluge textarea:invalid {
     color: red; }
+  .deluge input {
+    width: 100%; }
+  .deluge input:invalid {
+    color: red; }
```

Comment:
We should replace these `red`'s with `var(--tomato)`.

I think it's also worth combining these invalid states:
```
.deluge textarea:invalid,
.deluge input:invalid {
  color: var(--tomato);
}
```